src/geo/leiUtils.js

```diff
@@ -0,0 +1,38 @@
+import { runFetch, makeFilersUrl } from '../api.js'
+import { isEqual } from 'lodash'
+
+export function keepValidLeis(valid, selected) {
+  return selected.filter(s => valid[s])
+}
+
+export function filterLeis() {
+  const leis = this.state.leiDetails.leis
```

Comment:
I'm a little squeamish about having these functions as standalone when they are clearly used as methods on the Geography class (including referencing `this`. That said, I can appreciate wanting to break some code out of that file, which is getting pretty monstrous.

Reply:
They are bound in the constructor and it just looked nicer not passing state and setState to each filterLeis and fetchLeis call. 🤷‍♂ 

https://github.com/cfpb/hmda-data-browser/blob/filter-leis-by-geography/src/geo/Geography.jsx#L40-L41

Reply:
Yeah, I saw that they were bound, I just mean it's a but of a departure from the style of the existing codebase (but not necessarily a bad one!) Could make sense to pull out most like methods into their own modules sense we're binding them all in the constructor anyway. Food for thought